core/ir/x86/decode_table.c

```diff
@@ -3141,7 +3142,7 @@ const instr_info_t prefix_extensions[][12] = {
     {OP_movsd,  0xf20f1110, "movsd",  Wsd, xx, Vsd, xx, xx, mrm, x, END_LIST},
     {OP_vmovups,   0x0f1110, "vmovups", Wvs, xx, Vvs, xx, xx, mrm|vex, x, tevexw[0][0]},
     {MOD_EXT,    0xf30f1110, "(mod ext 10)", xx, xx, xx, xx, xx, mrm|vex, x, 10},
-    {OP_vmovupd, 0x660f1110, "vmovupd", Wvd, xx, Vvd, xx, xx, mrm|vex, x, tevexw[2][1]},
+    {OP_vmovupd, 0x660f1110, "vmovupd", Wvd, xx, Vvd, xx, xx, mrm|vex, x, tevexw[2][2]},
```

Comment:
Just trying a sanity check: so this was pointing at the entry for rex.w being set: so this requires rex.w to be set?  Should that mean its opcode should have OPCODE_MODRM set, which would be 0x40?  But it does not (only has 0x10)?  Am I missing something?

Reply:
This doesn't actually require REX.W=1 with the VEX prefix, it's a WIG instruction.

Reply:
I do wonder if there are tests somewhere for these instructions with C4 VEX prefixes and both values of W testing that the W is in fact IGnored. The binutils tests I see via git grep for vmovupd are all C5 VEX encodings.

Reply:
Seem to decode ok at least.

khuey@minbar:~/dev/dynamorio/obj$ ./clients/bin64/drdisas -syntax dr c5 fd 10 f4
 c5 fd 10 f4          vmovupd %ymm4 -> %ymm6
khuey@minbar:~/dev/dynamorio/obj$ ./clients/bin64/drdisas -syntax dr c4 e1 7d 10 f4
 c4 e1 7d 10 f4       vmovupd %ymm4 -> %ymm6
khuey@minbar:~/dev/dynamorio/obj$ ./clients/bin64/drdisas -syntax dr c4 e1 fd 10 f4
 c4 e1 fd 10 f4       vmovupd %ymm4 -> %ymm6